app/services/stepfunctions/RegularContributionsClient.scala

```diff
@@ -107,6 +106,12 @@ class RegularContributionsClient(
   }
 
   def status(jobId: String, requestId: UUID): EitherT[Future, RegularContributionError, StatusResponse] = {
+
+    def respondToClient(statusResponse: StatusResponse) = {
```

Comment:
Is it worth to try to make this function more generic and place it in a shared object?

Reply:
Hmm... the only thing that's really re-usable here is the specific log message so I'm not sure that there's much value in trying to make this more generic. It might be worth us creating a custom logging trait (or similar) which logs the request id by default though